src/server/server.js

```diff
@@ -0,0 +1,9 @@
+const http = require('http');
+const router = require('./router.js');
+
+const PORT = 8001;
```

Comment:
You could add a process.env.PORT here e.g. `const PORT  = process.env.PORT || 8001` 👍